flink/interpreter/src/main/scala/org/apache/zeppelin/flink/FlinkScalaInterpreter.scala

```diff
@@ -450,9 +450,9 @@ class FlinkScalaInterpreter(val properties: Properties) {
   }
 
   private def registerHiveCatalog(): Unit = {
-    val hiveConfDir =
-      properties.getOrDefault("HIVE_CONF_DIR", System.getenv("HIVE_CONF_DIR")).toString
-    if (hiveConfDir == null) {
+    val hiveConfDir = sys.env.getOrElse("HIVE_CONF_DIR",
+      properties.getProperty("HIVE_CONF_DIR", "")).toString
```

Comment:
The priority seems be changed between environment variable and property, is it intentional?